### PR TITLE
PLEASE DO NOT MERGE CI TEST Purpose(CRC installation)

### PR DESCRIPTION
**Pull request by flacatus**
This tests include installation of CRC in ci.centos

### CODE REVIEW

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

**Comment by flacatus** [comment]
[test]

File: olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.12.1/eclipse-che-preview-openshift.v7.12.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ metadata:
                 "chePostgresDb": ""
              },
              "auth": {
-                "openShiftoAuth": true,
+                "openShiftoAuth": false,
```
Comment (davidfestal) [inline olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.12.1/eclipse-che-preview-openshift.v7.12.1.clusterserviceversion.yaml:30]: Why are you changing this in the OLM packages ? Usually we should never change anything in those folders without creating a new nightly version first.

Comment (flacatus) [reply olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.12.1/eclipse-che-preview-openshift.v7.12.1.clusterserviceversion.yaml:30]: @davidfestal Is just for test purpose. I'm trying to install in Centos CI and run test updates. When I'll finish this task obviously I'll not change this files.

**Pull request closed without merging**